README.md

```diff
@@ -18,3 +18,14 @@ See its docs: https://godoc.org/inet.af/netaddr
 * merges net.IPAddr and net.IP (which the Go net package is a little torn between for legacy reasons)
 * ...
 * TODO: finish this list
+
+## Testing
+
+In addition to regular Go tests, netaddr uses fuzzing.
+To use:
+
+```
+$ go get -u github.com/dvyukov/go-fuzz/go-fuzz github.com/dvyukov/go-fuzz/go-fuzz-build
+$ go-fuzz-build && go-fuzz
```

Comment:
I feel like I've had to use `CGO_ENABLED=0 go-fuzz-build` for a long time on linux/amd64 to get things working. Have you run into that?

Reply:
Oh, I see https://github.com/dvyukov/go-fuzz/issues/101. I've not run into it. 2016 is a long time ago, and I re-worked large sections of go-fuzz-build to use go/packages. I wonder whether maybe this is fixed now. (Fingers crossed!)